org.syphon.Syphon.json

```
@@ -0,0 +1,96 @@
+    "sdk": "org.gnome.Sdk",
+    "command": "syphon",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download",
+        "--talk-name=org.freedesktop.Notifications",
+        "--system-talk-name=org.freedesktop.NetworkManager",
+    ],
+    "modules": [
+        {
+            "name": "libjsoncpp",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
+                "-DBUILD_SHARED_LIBS=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.7.6.tar.gz",
+                    "sha256": "07cf5d4f184394ec0a9aa657dd4c13ea682c52a1ab4da2fb176cb2d5501101e8"
+                }
+            ]
+        },
+        {
+            "name": "olm",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.matrix.org/matrix-org/olm.git",
+                    "tag": "3.2.8"
+                }
+            ]
+        },
+        {
+            "name": "syphon",
+            "buildsystem": "simple",
+            "only-arches": [
+                "x86_64"
+            ],
+            "build-commands": [
+                "mv syphon /app/syphon",
+                "chmod +x /app/syphon/syphon",
+                "install -Dm644 logo.svg /app/share/icons/hicolor/scalable/apps/org.syphon.Syphon.svg",
+                "mkdir -p /app/bin",
+                "ln -s /app/syphon/syphon /app/bin/syphon",
+                "install -Dm644 org.syphon.Syphon.desktop /app/share/applications/org.syphon.Syphon.desktop",
+                "install -Dm644 org.syphon.Syphon.metainfo.xml /app/share/appdata/org.syphon.Syphon.metainfo.xml"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/syphon-org/syphon/releases/download/0.2.3/syphon.linux.x64.tar.gz",
+                    "dest": "syphon",
+                    "sha256": "7b4a2f5929b04131eb9b74e28cf97a1ef70c3ac3e3224e41092f12ac42c7c6fa",
+                    "x-checker-data": {
+                        "type": "json",
+                        "is-main-source": true,
+                        "url": "https://api.github.com/repos/syphon-org/syphon/releases/latest",
+                        "version-query": ".tag_name",
+                        "tag-query": ".tag_name",
+                        "timestamp-query": ".published_at",
+                        "url-query": ".html_url"
```

The html url isn't a link is something like [this](https://github.com/syphon-org/syphon/releases/tag/0.2.5), I assume it isn't what you wanted.
What you did want is probably this:
```suggestion
                        "url-query": ".assets[] | select(.name==\"syphon.linux.x64.tar.gz\") | .browser_download_url"
```

---

Please, either fix the `x-checker-data` metadata or remove it.

---

Aha, sorry, forgot about this. I'll commit your suggestion.